src/definitions/modules/dropdown.js

```diff
@@ -671,7 +658,7 @@ $.fn.dropdown = function(parameters) {
               }
             }
             $menu
-              .on('mouseenter' + eventNamespace, selector.item, module.event.item.mouseenter)
+              .on((hasTouch ? 'touchstart' : 'mouseenter') + eventNamespace, selector.item, module.event.item.mouseenter)
               .on('mouseleave' + eventNamespace, selector.item, module.event.item.mouseleave)
```

Comment:
Is it OK not adding `touchend` to this line in a symmetric fashion ?

Reply:
I don't think so, because `mouseleave` would hide a possible submenu. This is ok when using a real mouse but unwanted when using touch devices (one would need to swipe between dropdown submenu entreies instead of touching each entry)
It also wasn't used before this PR (mabe because of the above mentioned reason). Only `mouseenter` had an additional touch event, so i only replace that one.

Reply:
@exoego Just tested for clarification and i am right, here is the proof: Submenus won't work anymore (because hidden when `touchend` is used instead of keeping `mouseleave`)
https://jsfiddle.net/d0snvrxb/1/show
![touchend_bad](https://user-images.githubusercontent.com/18379884/71075924-2b5b6b80-2185-11ea-965f-03c56669dd0c.gif)

Using only the changes of this PR (keep `mouseleave`), it works as before and expected
https://jsfiddle.net/d0snvrxb/show
![mouseleave_good](https://user-images.githubusercontent.com/18379884/71075938-344c3d00-2185-11ea-8714-85cd473e120a.gif)

Reply:
Understood